src/legacy/core_plugins/expressions/public/np_ready/public/render.ts

```diff
@@ -49,8 +49,11 @@ export class ExpressionRenderHandler {
     this.eventsSubject = new Rx.Subject();
     this.events$ = this.eventsSubject.asObservable().pipe(share());
 
-    this.renderSubject = new Rx.Subject();
-    this.render$ = this.renderSubject.asObservable().pipe(share());
+    this.renderSubject = new Rx.BehaviorSubject(null as RenderId | RenderError | null);
```

Comment:
When using expression service, render gets called before consumer have change to subscribe to render$ observable, which means that all sync error which are emitted in render() are swallowed.

By this change, whenever consumer subscribes to render$ it will get the last emitted value, unless render$ is in initial state (`null` )